src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -406,6 +406,383 @@
+                        {
+                            "Id": "AutoScaling.Enabled",
+                            "Value": true
+                        }
+                    ]
+                },
+                {
+                    "Id": "CpuTypeTargetUtilizationPercent",
+                    "Name": "CPU Target Utilization",
+                    "Description": "The target cpu utilization that triggers a scaling change.",
+                    "Type": "Double",
+                    "DefaultValue": 70,
+                    "AdvancedSetting": false,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AutoScaling.Enabled",
+                            "Value": true
+                        },
+                        {
+                            "Id": "AutoScaling.ScalingType",
+                            "Value": "Cpu"
+                        }
+                    ]
+                },
+                {
+                    "Id": "CpuTypeScaleInCooldownSeconds",
+                    "Name": "Scale in cooldown (seconds)",
+                    "Description": "The amount of time, in seconds, after a scale in activity completes before another scale in activity can start.",
+                    "Type": "Int",
+                    "DefaultValue": 300,
+                    "AdvancedSetting": false,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AutoScaling.Enabled",
+                            "Value": true
+                        },
+                        {
+                            "Id": "AutoScaling.ScalingType",
+                            "Value": "Cpu"
+                        }
+                    ]
+                },
+                {
+                    "Id": "CpuTypeScaleOutCooldownSeconds",
+                    "Name": "Scale out cooldown (seconds)",
+                    "Description": "The amount of time, in seconds, after a scale out activity completes before another scale out activity can start.",
+                    "Type": "Int",
+                    "DefaultValue": 300,
+                    "AdvancedSetting": false,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "AutoScaling.Enabled",
+                            "Value": true
+                        },
+                        {
+                            "Id": "AutoScaling.ScalingType",
+                            "Value": "Cpu"
+                        }
+                    ]
+                },
+                {
+                    "Id": "MemoryTypeTargetUtilizationPercent",
+                    "Name": "Memory Target Utilization",
+                    "Description": "The target memory utilization that triggers a scaling change.",
+                    "Type": "Double",
```

Comment:
Why is this a double and not an Int?

Reply:
https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ecs.CpuUtilizationScalingProps.html#targetutilizationpercent

underlying data structure is double

Reply:
Correct the service/cloudformation value is a double